packages/plugins/operators/operators-js/src/operators/shared/and.test.js

```diff
@@ -13,34 +13,69 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
+import { NodeParser, WebParser } from '@lowdefy/operators';
+import _and from './and.js';
 
-import and from './and.js';
+const operators = {
+  _and,
+};
 
 const location = 'location';
 
 test('_and false', () => {
-  expect(and({ params: [0, 0], location })).toEqual(false);
-  expect(and({ params: [0, 1], location })).toEqual(false);
-  expect(and({ params: [1, 2, 3, 0], location })).toEqual(false);
-  expect(and({ params: [false, false], location })).toEqual(false);
-  expect(and({ params: [false, true], location })).toEqual(false);
+  expect(_and({ params: [0, 0], location })).toEqual(false);
+  expect(_and({ params: [0, 1], location })).toEqual(false);
+  expect(_and({ params: [1, 2, 3, 0], location })).toEqual(false);
+  expect(_and({ params: [false, false], location })).toEqual(false);
+  expect(_and({ params: [false, true], location })).toEqual(false);
 });
 test('_and true', () => {
-  expect(and({ params: [1, 2], location })).toEqual(true);
-  expect(and({ params: [1, 2, 3], location })).toEqual(true);
-  expect(and({ params: [true, true], location })).toEqual(true);
+  expect(_and({ params: [1, 2], location })).toEqual(true);
+  expect(_and({ params: [1, 2, 3], location })).toEqual(true);
+  expect(_and({ params: [true, true], location })).toEqual(true);
 });
 test('_and errors', () => {
-  expect(() => and({ params: 'hello', location })).toThrow(
+  expect(() => _and({ params: 'hello', location })).toThrow(
     'Operator Error: _and takes an array type. Received: "hello" at location.'
   );
-  expect(() => and({ params: null, location })).toThrow(
+  expect(() => _and({ params: null, location })).toThrow(
     'Operator Error: _and takes an array type. Received: null at location.'
   );
-  expect(() => and({ params: true, location })).toThrow(
+  expect(() => _and({ params: true, location })).toThrow(
     'Operator Error: _and takes an array type. Received: true at location.'
   );
-  expect(() => and({ params: false, location })).toThrow(
+  expect(() => _and({ params: false, location })).toThrow(
     'Operator Error: _and takes an array type. Received: false at location.'
   );
 });
+
+test('_and calls NodeParser', async () => {
```

Comment:
We need a better name here. Maybe something like `_and evaluated in NodeParser` or `NodeParser integration test` or something